### DISCUSSION TITLE
Collapsible Sidebar

### DISCUSSION THREAD

**Discussion by RealityGamin** [laravel/nova-issues#5473] [Ideas & Feature Requests]
Can we please implement a features where the user can collapse the sidebar?

Sometimes it would be lovely to be able to do some work with that extra 15% space.

Something similar to what was created in this package but more smooth transitioning https://novapackages.com/packages/osama/nova-collapsible-sidebar

Thank you

**Comment by fedeisas** [comment]
Adding a toggle to collapse the sidebar menu would be a valuable feature. I couldn't find a clean solution to implement this as an external package. The Sentry sidebar serves as an excellent inspiration for this functionality.

Upon reviewing Nova's source code, it appears there's significant groundwork already in place for the mobile menu (`toggleMainMenu`). This existing code could be reused for a desktop version (maybe `toggleMainMenuDesktop`).

The new getter can handle the `lg:ml-60` class on `AppLayout`.

![image](https://github.com/laravel/nova-issues/assets/251675/e2c10e30-d111-4617-88b7-2e9869fdac0c)

Thanks!

**Reply by fedeisas** [reply]
@laravel-nova-team Do you have any opinions on this or suggestions for implementing this with a package? Thanks a lot!

**Reply by davidhemphill** [reply]
I think it's a great idea, but sadly I've not been involved with Nova since February! I'm sure the team has some great improvements coming soon. 😀

**Reply by fedeisas** [reply]
@davidhemphill thanks for the reply and sorry for the spam then :-)

**Comment by Denobraz** [comment]
We realy need this feature too!

**Comment by evelynDickinson** [comment]
This would be super helpful.

**Comment by nastoychev** [comment]
I think this will improve a lot the UI. And at all, this issue is an easy one ;)

**Comment by mttzzz** [comment]
i need it too
